scrapers/Andomark.yml

```
@@ -34,9 +74,73 @@ xPathScrapers:
             selector: //link[@rel="canonical"]/@href
             postProcess:
```

If we add an anchor here we can reuse the logic in the pro site scraper below
```suggestion
            postProcess: &studioPostProcess
```